package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "overlay-plugin",
+  "version": "0.0.0",
```

Comment:
Should this match the OverlayPlugin version (and be updated by scripts, etc)? I don't know what general practice is, but I do think it would be more understandable if these matched, e.g. if you know you're on x.y.z OverlayPlugin then you should use x.y.z npm version.

Reply:
it depends. If you don't make breaking change frequently in OverlayPlugin, I think we only need to keep them have save major version. like npm package v0 for overlay plugin v0